src/main/cpp/VideoState.cpp

```diff
@@ -1075,27 +1065,25 @@ int VideoState::stream_start() {
 		goto fail;
 	}
 
-	if (genpts)
+	if (ENABLE_GENERATE_PTS) {
```

Comment:
Same here (see VideoState.h comment on line 75)

Reply:
Changed to use const